tests/TestApp/AdalDesktopTestApp/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="3.13.4" targetFramework="net46" />
+</packages>
```

Comment:
This file should never be there because the test app project uses the project references and not a shipped nuget.